docs/elemental_behind_proxy.md

```diff
@@ -0,0 +1,77 @@
+title: ''
+---
+
+import RegistrationProxy from "!!raw-loader!@site/examples/proxy/registration-proxy.yaml"
+import SeedimageProxy from "!!raw-loader!@site/examples/proxy/seedimage-proxy.yaml"
+import ClusterProxy from "!!raw-loader!@site/examples/proxy/cluster-proxy.yaml"
+
+## Introduction
+
+In a lot of enterprise environments, servers or VMs running on premise do not have direct Internet access, but must connect to external services through a HTTP(S) proxy for security reasons. This tutorial shows step by step how to set up an Elemental deployment in such an environment.
+
+First of all, we will not cover Rancher installation behind a proxy, it is a different use case and you can find the documentation [here](https://ranchermanager.docs.rancher.com/pages-for-subheaders/rancher-behind-an-http-proxy).
+
+:::info info
+For this documentation, we assume you are using a SUSE family system (like Elemental Teal), so proxy settings have to be written in `/etc/sysconfig/proxy`.
+:::
+
+Proxy settings have to be configured in 3 places:
+- Machine Registration Endpoint
+- SeedImage resource
+- Elemental cluster configuration
+
+
+## Machine Registration Endpoint
+The elemental-system-agent needs proxy settings to reach the Rancher Manager.
+To achieve that, we need to fill the cloud-init section of the Machine Registration Endpoint:
+
+You can do it either with [UI](https://elemental.docs.rancher.com/quickstart-ui#add-a-machine-registration-endpoint) or [CLI](https://elemental.docs.rancher.com/quickstart-cli#prepare-your-kubernetes-resources).
+
+<Tabs>
+<TabItem value="cliRegistration" label="CLI" default>
+<CodeBlock language="yaml" title="registration.yaml" showLineNumbers>{RegistrationProxy}</CodeBlock>
+</TabItem>
+<TabItem value="uiRegistration" label="UI" default>
+
+![Add proxy settings in Machine Registration](images/proxy-settings-machine-registration-ui.png)
+</TabItem>
+</Tabs>
+
+
+
+## Elemental-register
+
+[Elemental-register](https://elemental.docs.rancher.com/architecture/#elemental-register-client) is the first communication endpoint between the new host and Rancher Manager, this is the first place where proxy settings could be needed.
+
+:::warning warning
+At the time of writing, that's only possible to configure proxy settings for the ISO in CLI, nothing is implemented in the UI yet ( and we do not know if it will be, ask for it!)
+:::
+
+The process happens when you boot your Elemental ISO for the first time, in order to configure proxy for it, you have to include a cloud-init definition in the ISO.
+To do that, you have to create a [SeedImage](https://elemental.docs.rancher.com/seedimage-reference/) definition.
+
+<CodeBlock language="yaml" title="seedimage.yaml" showLineNumbers>{SeedimageProxy}</CodeBlock>
+
+Apply the YAML with kubectl and then, print your SeedImage definition to get the URL to download it:
+
+```bash showLineNumbers
+kubectl get seedimage <seed_image_name> -n <namespace> -o yaml
```

Comment:
As the description before the codeblock mentions an apply, can you add it here too?